GitTfs/Commands/Checkout.cs

```diff
@@ -43,7 +43,8 @@ public int Run(string id)
             long changesetId;
             if(!long.TryParse(id, out changesetId))
                 throw new GitTfsException("error: wrong format for changeset id...");
-            var sha = _globals.Repository.FindCommitHashByChangesetId(changesetId);
+            //TODO
+            var sha = _globals.Repository.FindCommitHashByChangesetId(changesetId, "TODO");
```

Comment:
Make FindCommitByChangesetIdS, which will find all matching commits on ALL branches (remove `1c.Message.Contains(tfsPath)` and show warning or prompt to the user if there are more than one commit/branch.